Generator/Generator/Arguments.swift

```diff
@@ -258,3 +262,49 @@ func generateArgPrepare (isVararg: Bool, _ args: [JGodotArgument], methodHasRetu
     }
     return (body, withUnsafeCallNestLevel)
 }
+
+#else
+
+func generateArgPrepare (isVararg: Bool, _ args: [JGodotArgument], methodHasReturn: Bool) -> (String, Int) {
+    var body = ""
+    var withUnsafeCallNestLevel = 0
+    let retFromWith = methodHasReturn ? "return " : ""
+    
+    // TODO: this case should get the same treatment as a second branch.
+    if isVararg {
+        body += generateCopies (args)
+        body += "var args: [UnsafeRawPointer?] = []\n"
+        if isVararg {
+            body += "let cptr = UnsafeMutableBufferPointer<Variant.ContentType>.allocate(capacity: arguments.count)\n"
```

Comment:
This is the test duplication, and there is another one below from the original code.

Reply:
Fixed